CodeSnippetsReflection.OpenAPI/LanguageGenerators/PhpGenerator.cs

```diff
@@ -335,26 +332,36 @@ private static void WriteMapValue(StringBuilder payloadSb, string propertyAssign
         }
 
         var childPosition = 0;
-        indentManager.Indent(2);
+        indentManager.Indent();
         foreach (var child in currentProperty.Children)
         {
+            var payLoad = new StringBuilder();
             payloadSb.Append($"{indentManager.GetIndent()}\'{child.Name}\' => ");
             CodeProperty p2 = child;
             if (p2.PropertyType == PropertyType.Object)
             {
                 p2.PropertyType = PropertyType.Map;
             }
-            WriteCodeProperty(propertyAssignment, payloadSb, currentProperty, p2, indentManager, ++childPosition, default, true);
-            payloadSb.AppendLine();
+            WriteCodeProperty(propertyAssignment, payLoad, currentProperty, p2, indentManager, ++childPosition, default, true);
+            payLoad.AppendLine();
+            payloadSb.AppendLine(payLoad.ToString().Trim(' ', '\t', '\n'));
```

Comment:
No need to pass params to Trim if what you are trimming is whitespace. 
https://learn.microsoft.com/en-us/dotnet/api/system.string.trim?view=net-7.0
```suggestion
            payloadSb.AppendLine(payLoad.ToString().Trim());
```